lib/unleash/constraint.rb

```diff
@@ -92,6 +94,12 @@ def matches_constraint?(context)
         false
       end
 
+      # when the operator is NOT_IN and there is no data, return true. In all other cases the operator doesn't match.
+      return self.operator == :NOT_IN unless context.property?(self.context_name)
```

Comment:
```suggestion
      return self.operator == :NOT_IN unless context.include?(self.context_name)
```

Reply:
this one is on an outdated thing, this is fixed